src/app/modules/project/overview/overview.component.ts

```diff
@@ -309,27 +280,14 @@ export class OverviewComponent implements OnInit, AfterContentInit {
    * Method to handle value changes of the sort form.
    * @param value the value of the form.
    */
-  private onSortFormValueChange(value: SortFormResult): void {
-    if (value == null || value.type == null || value.type === 'null') {
+  public onSortFormValueChange(value: string): void {
+    if (value == null) {
       return;
     }
-    this.currentSortType = value.type;
-    this.currentSortDirection = value.direction;
-    this.onInternalQueryChange();
-  }
-
-  /**
-   * Method to handle the value changes of the highlight form.
-   * @param value the value of the highlight form.
-   */
-  private onHighlightFormValueChanges(value: number): void {
-    // Case the value since for some reason it is always returned as a string.
-    const convertedIndex = +value;
-    const foundOption = this.highlightSelectOptions[convertedIndex];
-    if (foundOption == null) {
-      return;
-    }
-    this.currentOnlyHighlightedProjects = foundOption.value;
+    this.currentSortType = value.split(',')[0];
+    this.currentSortDirection = value.split(',')[1];
+    console.log(`value.type: ${this.currentSortType}`);
+    console.log(`value.direction: ${this.currentSortDirection}`);
```

Comment:
Is this something that should get logged?

Reply:
Deze heb ik er uit gehaald :)